test/transaction.js

```diff
@@ -122,6 +122,35 @@ describe('Transaction', function () {
     })
   })
 
+  describe('fromBuffer/fromHex for Testnet Dash', function () {
```

Comment:
We also need a test that generates a valid signed transaction (a multi sig test would be best) since we use them as example on how to use the library.

Reply:
@argjv Done